### PR TITLE
Update es.json

### PR DESCRIPTION
**Pull request by antoniormesa**
Add es.json for spanish language.

### CODE REVIEW

File: src/assets/i18n/es.json [hunk 1]
```diff
@@ -90,11 +90,11 @@
 		"languages": {
 			"cnil_translations": "Traducciones del CNIL",
 			"community_translations": "Traducciones de la comunidad",
-			"cz": "Česky",
+			"cz": "Cesky",
```
Comment (brunto) [inline src/assets/i18n/es.json]: Please keep the original traduction.

File: src/assets/i18n/es.json [hunk 2]
```diff
@@ -90,11 +90,11 @@
...
 			"de": "Deutsch",
 			"dk": "Dansk",
-			"dpa_translations": "TO BE TRANSLATED - Translation revised by European DPA",
-			"el": "Ελληνικά",
+			"dpa_translations": "Traducción revisada por la Autoridad de Protección de Datos Europea",
+			"el": "????????",
```
Comment (brunto) [inline src/assets/i18n/es.json]: Please keep the original traduction.

File: src/assets/i18n/es.json [hunk 3]
```diff
@@ -109,13 +109,13 @@
 			"official_translation": "Versión oficial",
 			"pl": "Polski",
 			"pt": "Português",
-			"ro": "Română",
-			"sl": "Slovenščina"
+			"ro": "Româna",
+			"sl": "Slovenšcina"
```
Comment (brunto) [inline src/assets/i18n/es.json]: Please keep the original traduction.

File: src/assets/i18n/es.json [hunk 4]
```diff
@@ -397,117 +397,117 @@
...
+			},
+			" PIA_DEF_VIOL " : {
+				" descripción " : " Infracción de seguridad que conduce a la destrucción accidental o ilegal, pérdida, alteración, divulgación no autorizada o acceso a datos personales transmitidos, almacenados o procesados ??de otra manera " ,
+				" nombre " : " Violación de datos personales " ,
+				" fuente " : " GDPR Art. 4.12 "
+			},
+			" PIA_DEF_VRA " : {
+				" Descripción " : ". Probabilidad expresa la posibilidad de ocurrencia de un riesgo que depende principalmente del nivel de las vulnerabilidades de los activos de apoyo cuando están bajo amenaza y el nivel de las capacidades de las fuentes de riesgo de explotación de los mismos " ,
+				" nombre " : " Probabilidad " ,
+				" fuente " : " PIA1 página 6 "
```
Comment (brunto) [inline src/assets/i18n/es.json]: Could you remove the extra spaces?

Comment (antoniormesa) [reply src/assets/i18n/es.json]: Sorry for time to answer. I'm on it.

File: src/assets/i18n/es.json (hunk 1 above)
Comment (antoniormesa) [reply src/assets/i18n/es.json]: Ok

**Comment by antoniormesa** [comment]
I have reviewed conflicts and I have also removed extra spaces you had commented on.
Sorry for any mistake that I still don't work correctly in GitHub platform

File: src/assets/i18n/es.json (hunk 4 above)
Comment (antoniormesa) [reply src/assets/i18n/es.json]: .

**Comment by antoniormesa** [comment]
There are new sections to translate for PIA 2.3, which will need to be updated when this changes are validated.

File: src/assets/i18n/es.json (hunk 1 above)
Comment (brunto) [reply src/assets/i18n/es.json:119]: Could you keep the original traduction? It must be "Česky". thx

File: src/assets/i18n/es.json [hunk 5]
```diff
@@ -116,7 +116,7 @@
 		"languages": {
 			"cnil_translations": "Traducciones del CNIL",
 			"community_translations": "Traducciones de la comunidad",
-			"cz": "Česky",
+			"cz": "Cesky",
```
Comment (brunto) [inline src/assets/i18n/es.json:119]:
```suggestion
			"cz": "Česky",
```

Comment (antoniormesa) [reply src/assets/i18n/es.json]: Thanks for resolving this.

**Comment by kosmas58** [comment]
Your pull request contsins several error which I have coorected in a New pull request. It is also appropriate that you Removed the authorship of ylachgar for the spanisch translation. I have also coorected that.

Regards, Kosmas

**Pull request merged**